### PR TITLE
Changes to USBHID.md

### PR DESCRIPTION
**Pull request by mrcoulter45**
- modify USBHID example by adding a helper Python script

### CODE REVIEW

File: docs/reference/api/usb/USBHID.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
 
 <span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_h_i_d.png)<span>USBHID class hierarchy</span></span>
 
-You can use the USBHID class to send and receive messages over USB. For example, you can define your own protocol and communicate between your computer and Mbed with all capabilities of USB communication. To use USBHID, you need a script running on the host side (computer). For example, on a 32-bit Windows 7 machine, you can use [pywinusb](https://github.com/rene-aguirre/pywinusb).
+You can use the USBHID class to turn an Mbed board into an HID (Human Interface Device) that can send and receive messages over USB. For example, you can define your own protocol and communicate between your computer and Mbed with all capabilities of USB communication. To use USBHID, you need a script running on the host side (computer). For example, on a 32-bit Windows 7 machine, you can use [pywinusb](https://github.com/rene-aguirre/pywinusb). For convenience, there is a Python script attached below that uses pywinusb that can be run on the host computer to send and receive data from the Mbed board.
```
Comment (c1728p9) [inline docs/reference/api/usb/USBHID.md]: `on a 32-bit Windows 7 machine` is a bit specific. Could this be replaced with just `Windows machine`?

**Comment by AnotherButler** [comment]
Note: When USB merges to master, we'll add the examples to teams and make this document live.

File: docs/reference/api/usb/USBHID.md (hunk 1 above)
Comment (mrcoulter45) [reply docs/reference/api/usb/USBHID.md]: fixed

File: docs/reference/api/usb/USBHID.md [hunk 2]
```diff
@@ -40,14 +40,54 @@ int main(void) {
...
                 pc.printf("%d ", recv_report.data[i]);
             }
             pc.printf("\r\n");
         }
     }
 }
-```
+```   
+##### USBHID.py   
+To use this script, first flash the Mbed board with the code above. Connect the USBPhy to the host computer. Then run this script. The script will send 8 bytes of data (1 2 3 4 5 6 7 8) to the Mbed board and will read and print the data sent to the host computer by the Mbed board.
```
Comment (c1728p9) [inline docs/reference/api/usb/USBHID.md]: You could probably use `the target's USB` in place of `USBPhy`

**Comment by AnotherButler** [comment]
I swear I edited it, but now I don't see my changes

**Pull request merged**